project_parameters.Config.yaml

```
@@ -105,10 +105,10 @@ use_min.diff.med_annotation_module: "NO" # Options: "YES" or "NO" (by default)
 assay_annotation_module: "RNA"  # Options: "RNA" (by default) or "RNA_SoupX"
 
 gene_markers_dir_annotation_module: "./" # path to gene markers dir
-gene_markers_file_name_annotation_module: "<filename>.tsv" # File name with the gene markers. It needs to be in `tsv` format.
+gene_markers_file_name_annotation_module: "<filename>.tsv" # File name with the gene markers. It needs to be in `tsv` format. Please ensure cell type names exist and match the ones in the `.figures/palettes/cell_types_palette.tsv`.
 #cluster_name: "seurat_clusters"
-clustering_gene_markers_dir_annotation_module: "./sc-rna-seq-snap/analyses/cluster-cell-calling/results/02_find_markers" # path to gene markers dir as generated from the `cluster_cell_calling` 
 method_annotation_module: "all" # Options: "all" (for using singler and a list of gene markers), "singler" (for using only singler method for both broad and fine annotation)
+gene_name_convention_update_annotation_module: "NO" # Options: "YES" or "NO" (by default)
```

I think this parameter should be explained a bit more for the user (either here in the yaml or in the README).

---

Please provide explanation to be added in the yaml as documentation.

---

```suggestion
gene_name_convention_update_annotation_module: "NO" # Options: "YES" or "NO" (by default), set to "YES" if gene names in gene markers file do not match naming convention of genes in reference genome.
```

Or something similar to this.

---

Done, see 9e4e06ab1f36ea425365eea0bae0c931f31faca3.